msprime/tables.py

```diff
@@ -603,15 +600,14 @@ def sort_tables(*args, **kwargs):
     are not affected. This parameter is provided to allow for efficient sorting
     when the user knows that the edges up to a given index are already sorted.
 
-    .. todo:: Update this documentation to describe the keyword arguments and
-       combinations that are allowed.
-
-    :param NodeTable nodes:
-    :param EdgeTable edges:
-    :param MigrationTable migrations:
-    :param SiteTable sites:
-    :param MutationTable mutations:
-    :param int edge_start: The index in the edge table where sorting starts.
+    :param NodeTable nodes (required).
```

Comment:
I don't this this is valid sphinx markup. It should be something like
```
:param NodeTable nodes: The tree sequence nodes (required).
```
See http://www.sphinx-doc.org/en/stable/domains.html for examples of the required markup formats.